sentry/src/test/java/io/sentry/GsonSerializerTest.kt

```diff
@@ -556,6 +563,21 @@ class GsonSerializerTest {
         assertEquals(mapOf("name" to "value"), span.tags)
     }
 
+    @Test
+    fun `serializes span data`() {
+        val sentrySpan = SentrySpan(createSpan() as Span, mapOf("data1" to "value1"))
+
+        val stringWriter = StringWriter()
+        fixture.serializer.serialize(sentrySpan, stringWriter)
```

Comment:
should we use one of the `serializeToString` methods?

Reply:
Yep, makes sense